contents/articles/2016-07_new-headset-color-dye-process/index.md

```diff
@@ -0,0 +1,14 @@
+---
+ title: New Headset Color and Dye Process
+ date: Tue Jul 07 2016 09:47:48 GMT+0700 (ICT)
+ author: Pupil Dev Team
+ subtitle: We have been experimenting with an in-house dyeing process to perfect the finish of our headsets...
+ ---
+
+<div class="Feature-video-container-16by9">
+  <iframe class="Feature-video" src="https://www.youtube.com/embed/f8BM9y6Yz4A" frameborder="0" allowfullscreen></iframe>
+</div>
+
+<img src="../../../../media/images/blog/dmk-headset.jpg" class='Feature-image' alt="DMK Headset">
+
+We have been experimenting with an in-house dyeing process to perfect the finish of our headsets. It involves careful measurement for the ideal dye to surface area of parts ratio, temperature control to produce the most saturated color, and a sealing process for product protection. Recently, we began experimenting with color alternatives to the current black headset. We are working to perfect this process for a vibrant cobalt color.
```

Comment:
Remove line `Recently...` at the end of this paragraph and add this two sentences above the image:
`We are experimenting with color alternatives to the current black Pupil headset. We have tried out a wide variety of colors, and there are a lot of great options, but the Pupil Dev Team really likes this vibrant cobalt color.`

The second paragraph should start:
`Our in-house dyeing process involves  careful...`